src/Form/CronJobDisableForm.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\ultimate_cron\Form\CronJobDisableForm.
+ */
+
+namespace Drupal\ultimate_cron\Form;
+
+use Drupal\Core\Entity\EntityConfirmFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+
+class CronJobDisableForm extends EntityConfirmFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getQuestion() {
+    return $this->t('Do you really want to disable @cronjob_id cron job?', array(
+      '@cronjob_id' => $this->getEntity()->label(),
+    ));
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getDescription() {
+    return $this->t('This cron job will not be able to be run.');
```

Comment:
I'd write "This cron job will no longer be executed" instead. It's not about being able to, it's about explicitly disabling it.